test/plots/segmentPlotTests.ts

```diff
@@ -2,159 +2,168 @@
-      plot.addDataset(new Plottable.Dataset([data[0]])).renderTo(svg);
-      renderArea = (<any> plot)._renderArea;
-      let lineSelection = d3.select(renderArea.selectAll("line")[0][0]);
-      assert.strictEqual(+lineSelection.attr("x1"), 62.5, "x1 is correct");
-      assert.strictEqual(+lineSelection.attr("x2"), 437.5, "x2 is correct");
-      assert.strictEqual(+lineSelection.attr("y1"), 437.5, "y1 is correct");
-      assert.strictEqual(+lineSelection.attr("y2"), 62.5, "y2 is correct");
-      svg.remove();
-    });
+    describe("SegmentPlot - basics", () => {
+      let svg: d3.Selection<void>;
+      let xScale: Plottable.Scales.Linear;
+      let yScale: Plottable.Scales.Linear;
+      let renderArea: d3.Selection<void>;
+      let data = [
+        { x: 1, y: 1, x2: 4, y2: 4 },
+        { x: 2, y: 2, x2: 3, y2: 5 },
+        { x: 3, y: 3, x2: 5, y2: 2 }
+      ];
 
-    it("renders vertical lines when x2 is not set", () => {
-      let plot = new Plottable.Plots.Segment();
-      plot.x(function(d) { return d.x; }, xScale);
-      plot.y(function(d) { return d.y; }, yScale);
-      plot.y2(function(d) { return d.y2; });
-      plot.addDataset(new Plottable.Dataset(data)).renderTo(svg);
-      renderArea = (<any> plot)._renderArea;
-      renderArea.selectAll("line")[0].forEach((line) => {
-      let lineSelection = d3.select(line);
-      assert.strictEqual(lineSelection.attr("x1"), lineSelection.attr("x2"), "line is vertical");
+      beforeEach(() => {
+        svg = TestMethods.generateSVG(500, 500);
+        xScale = new Plottable.Scales.Linear();
+        yScale = new Plottable.Scales.Linear();
       });
-      svg.remove();
-    });
 
-    it("renders horizontal lines when y2 is not set", () => {
-      let plot = new Plottable.Plots.Segment();
-      plot.x(function(d) { return d.x; }, xScale);
-      plot.x2(function(d) { return d.x2; });
-      plot.y(function(d) { return d.y; }, yScale);
-      plot.addDataset(new Plottable.Dataset(data)).renderTo(svg);
-      renderArea = (<any> plot)._renderArea;
-      renderArea.selectAll("line")[0].forEach((line) => {
-      let lineSelection = d3.select(line);
-      assert.strictEqual(lineSelection.attr("y1"), lineSelection.attr("y2"), "line is horizontal");
+      afterEach(() => {
+        svg.remove();
       });
-      svg.remove();
-    });
 
-    it("autorangeMode(\"x\")", () => {
-      let staggeredData = [
-        { y: 0, x: 0, x2: 1 },
-        { y: 1, x: 1, x2: 2 }
-      ];
-      xScale.padProportion(0);
+      it("renders a line properly", () => {
+        let plot = new Plottable.Plots.Segment();
+        plot.x((d) => d.x, xScale);
+        plot.x2((d) => d.x2);
+        plot.y((d) => d.y, yScale);
+        plot.y2((d) => d.y2);
+        plot.addDataset(new Plottable.Dataset([data[0]]));
+        plot.renderTo(svg);
+        renderArea = (<any> plot)._renderArea;
+        let lineSelection = d3.select(renderArea.selectAll("line")[0][0]);
```

Comment:
Probably better to just use `content()`:
```Typescript
let lineSelection = plot.content().select("line");
```

Probably should also assert that there is only one line.